data_services/deploy/ansible/roles/geonode/tasks/main.yml

```diff
@@ -44,7 +44,7 @@
   copy: src=known_hosts dest={{ssh_dir}}/known_hosts
 
 - name: checkout latest web app code
-  git: repo={{code_repository}} version={{app_version}} dest={{app_code_dir}}/{{app_name}} force=yes
+  git: repo={{code_repository}} version=2.4.x dest={{app_code_dir}}/{{app_name}} force=yes
```

Comment:
Sorry, I overlooked that; app_version is the better place to set branch to 2.4.x.  I updated the code and also added several more data's processors to run (gdacs and usgs_quakes).



> On Feb 5, 2016, at 10:12 PM, Aashish Chaudhary <notifications@github.com> wrote:
> 
> In data_services/deploy/ansible/roles/geonode/tasks/main.yml <https://github.com/Kitware/minerva/pull/285#discussion_r52096840>:
> 
> > @@ -44,7 +44,7 @@
> >    copy: src=known_hosts dest={{ssh_dir}}/known_hosts
> >  
> >  - name: checkout latest web app code
> > -  git: repo={{code_repository}} version={{app_version}} dest={{app_code_dir}}/{{app_name}} force=yes
> > +  git: repo={{code_repository}} version=2.4.x dest={{app_code_dir}}/{{app_name}} force=yes
> Sounds good so is the app_version is useless>?
> 
> —
> 


> Reply to this email directly or view it on GitHub <https://github.com/Kitware/minerva/pull/285/files#r52096840>.
>